2015/proposals/banaszkiewicz-piotr-amy.md

```diff
@@ -0,0 +1,214 @@
+# Enhance Amy, a workshop-management platform for Software Carpentry
+
+## Abstract
+
+The number of Software Carpentry's workshops run weekly dynamically grows. All
+the management for these workshops is currently being done by 1-2 persons and
+in very rough conditions (via files and text notes).  That's why Amy was
+created in December 2014: to help Software Carpentry admins schedule
+workshops, assign instructors to workshops, and even coordinate training for
+new instructors.
+
+The aim of this project is to continue enhancing Amy.  Many people
+[suggested great ideas](https://github.com/swcarpentry/amy/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement+-label%3Aessential) for Amy,
+some even [implemented them](https://github.com/swcarpentry/amy/pulls?q=is%3Apr+is%3Aopen+sort%3Acreated-asc), but not much was actually
+merged in -- and, as a result, in March
+[we can see a drop in new commits](https://github.com/swcarpentry/amy/graphs/contributors).
+
+## Features
+
+Integration that will allow Amy admins to easy control workshop-related
+services:
+
+* integration with Eventbrite
+    * checking if event's registration exists
```

Comment:
I'd rather drop the “ing”s and make this “check if …” “bulk-load …”, “create …”.  But that's probably because both [Git commit summaries][1] and [Python docstrings][2] should be imperitive.

[1]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
[2]: https://www.python.org/dev/peps/pep-0257/#one-line-docstrings